leveldb/sst/db.go

```diff
@@ -148,10 +171,10 @@ func BuildSSTable(f *os.File, memTable *skiplist.SkipList, tombstones *skiplist.
 	if _, err := f.Seek(0, io.SeekStart); err != nil {
 		return nil, err
 	}
-	if err := encoding.WriteInt64(f, directoryOffset); err != nil {
+	if err := encoding.WriteUint64(f, uint64(directoryOffset)); err != nil {
```

Comment:
Don't love all of this but haven't yet figured out a nice way to read bytes as int64